pkg/ddc/thin/referencedataset/cm.go

```diff
@@ -129,6 +129,9 @@ func createConfigMapForRefDataset(client client.Client, refDataset *datav1alpha1
 		if err != nil {
 			return err
 		}
+	case common.EACRuntime:
+		// TODO: EACRuntime needs worker-endpoint configmap which should be synced timely for ECI mode.
```

Comment:
Will you add logging for `case common.EACRuntime:` ?

Reply:
Done